docs/usage.md

```diff
@@ -91,3 +93,143 @@ Find and set the options which fit to your demands, it is also an option to chan
 | DietPi-RAMlog #1 | RAM | last hour | volatile, i.e. not saved to disk |
 | DietPi-RAMlog #2 | RAM | long term | stored, i.e. hourly saved to disk |
 | Full logging | disk | long term | stored, i.e. immediately saved to disk <br>(with Rsyslog and Logrotate)|
+
+---
+
+## How to do an automatic base installation at first boot (DietPi-Automation)
+
+DietPi offers the option for an automatic first boot installation. Normally, during the first system boot there is an installation procedure which sets up your system initially. The steps described in the section ["First logon on DietPi"](http://localhost:8000/docs/install/#4-first-logon-on-dietpi) are then conducted.
+
+![dietpi-login01](assets/images/dietpi-login01.jpg){: width="640" height="371" loading="lazy"}
```

Comment:
Use the image alt tag (text) to describe what can be seen on the image. Such kind of key words do not make sense to anyone who does not see the image, screen readers, blind people etc. If it is purely decorative, leave it completely empty, so also screen readers will not read out some cryptic key words that have no meaning and benefit to visitors 😉.

Reply:
Ok, changed to "DietPi first login - licence aggreement".
We then would need to check all graphics in the documentation to have a better figure description.